static/js/src/advantage/react/components/Subscriptions/Content/Content.test.tsx

```diff
@@ -16,12 +16,28 @@ describe("Content", () => {
     queryClient.setQueryData("userSubscriptions", [contract]);
   });
 
-  it("renders", () => {
+  it("displays a spinner when loading", () => {
+    // Remove the queries so that the query starts loading.
+    queryClient.removeQueries("userSubscriptions");
     const wrapper = mount(
       <QueryClientProvider client={queryClient}>
         <Content />
       </QueryClientProvider>
     );
+    expect(wrapper.find("[data-test='initial-load'] Spinner").exists()).toBe(
+      true
+    );
```

Comment:
Could I convince you to use `@testing-library/react` for this test instead? 😄

Reply:
(totally optional) 😉

Reply:
This might be out of scope of this PR as we should convert the whole file over vs just one/some of the tests.

Reply:
Yeah, I'm nearing the end of this ua table section of the site which has all been built with Enzyme, so I don't think this would be the right time to introduce a new testing lib.